rewrites.js

```diff
@@ -239,6 +243,10 @@ exports.REWRITES = [
     source: `/embed/:collectiveSlug/donate/:step(${contributionFlowSteps})?`,
     destination: '/embed/contribution-flow',
   },
+  {
+    source: `/embed/:collectiveSlug/:verb(donate)/:paymentMethod(crypto)/:step(${contributionFlowSteps})?`,
+    destination: '/embed/contribution-flow',
+  },
```

Comment:
Why do we need the previous one if we have this?

Reply:
You are right, we can combine both of the routes to a single one. I've done that. 👍🏽